containers/README.md

```diff
@@ -1,4 +1,14 @@
-# Docker files for EMG Viral Pipeline
+# Containers for VIRify
+
+## Container registry
+
+The containers used on this pipeline are stored under the [Microbiome Informatics]() (Quay organization)[https://quay.io/repository/microbiome-informatics/].
+
+## Build process
+
+The build process takes care of tagging the images using the LABELS on the Dockerfiles of the tools.
```

Comment:
yeah, as said in the other PR you could use the LABEL + md5 sum of the Dockerfile for tagging. Then you can have multiple containers with the same tool version but also with other changes (added dependencies, ...)

Reply:
Yeah... that is a good idea -> https://github.com/EBI-Metagenomics/emg-viral-pipeline/issues/85